apps/web/src/components/TrendingSuggestions/TrendingSuggestions.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { useQuery } from 'react-query';
+import fetchTrendingSuggestions from './TrendingSuggestions.api';
+import TrendingSuggestionList from '../TrendingSuggestionList/TrendingSuggestionList';
+
+const TrendingSuggestions = () => {
+  const { data, isLoading } = useQuery(
+    ['trending-suggestions'],
+    () => {
+      const response = fetchTrendingSuggestions();
+      return response;
+    },
+    { staleTime: Infinity }
+  );
+
+  // placeholder func, will be removed when we add react router + refactor
+  const handleClick = () => {
+    console.log('temp');
+  };
+
+  return (
+    <>
+      {data && (
```

Comment:
Won't this disable the 'loading' UI of your `TrendingSuggestionList` since the component will only render if the data is truthy, hence already loaded?